sdks/python/apache_beam/testing/synthetic_pipeline.py

```diff
@@ -496,7 +607,9 @@ def parse_args(args):
            '    Defaults to 0.'
            '(3) An integer "output_records_per_input_record". Defaults to 1.'
            '(4) A float "output_filter_ratio" in the range [0, 1] . '
-           '    Defaults to 0.')
+           '    Defaults to 0.'
+           '(5) A bool "splittable" that defaults to false.'
+           '(6) A integer "initial_splitting_num_bundles". Defaults to 2.')
```

Comment:
How about we do something like 8, so that its larger then splitting that LS would provide.